test/stripe/stripe_client_test.rb

```diff
@@ -1418,6 +1418,7 @@ class StripeClientTest < Test::Unit::TestCase
         assert_equal(200, event.http_status)
         assert(event.duration.positive?)
         assert_equal(0, event.num_retries)
+        assert_equal('req_123', event.request_id)
```

Comment:
```suggestion
        assert_equal("req_123", event.request_id)
```

Reply:
@richardm-stripe - Rubocop prefers double quotes over singles

```
test/stripe/instrumentation_test.rb:67:23: C: Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
          request_id: 'req_123',
                      ^^^^^^^^^

210 files inspected, 1 offense detected
```